### DISCUSSION TITLE
Some thoughts on detect resolution and bit rate

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#15168] [Ask A Question]
I have been using 704x480 as my detect resolution for over 2 years now and have been very happy with its performance at detecting people within 25 feet or so my Dahua cameras (3.6mm and 2.8mm). I know the documentation provides an example of 1280x720 being used as the detect resolution, but I think this would be overkill for my situation. I am not trying to detect people on the other side of the street, or even people hanging out next door. My primary goal with person detection is to turn on the sirens and flashing blue/red lights in my cameras if someone comes within 6 feet of my 1st floor windows.

Here's the view from one of my cameras at 704x480. As you can see, my house sits back from the street. Sometimes the system fails to identify people walking passed on the sidewalk but I think this is because a retaining wall obscures people's legs as they walk by. That is okay with me because the system will generally detect them if they climb up on the wall or yard. 

![image](https://github.com/user-attachments/assets/ca5bf5d5-3f07-4e31-b040-0fb5a74b037a)

I recently increased the bit rate from something around 192 Kb/s to 768 Kb/s on the detect stream and lowered the i frame interval from 10 to 5 fps to make the camera's fps. Does anyone have any insight into whether this will improve detection? It is certainly more taxing on the cpu.

Also, is it worth investing time to experiment with shutter speed, and other camera settings to try to improve person detection before I start collecting false positives and false negatives to feed to Frigate+?

**Comment by Cold-Lemonade** [comment]
Also, @blakeblackshear @NickM-27 @hawkeye217 and everyone else who has contributed to creating Frigate, I want to express my gratitude. It has helped make my neighborhood safer according to the local police.

**Comment by hawkeye217** [comment]
Thanks for your kind words, we're glad Frigate has been useful for you.

Your iframe interval matters more for live view than detection. A higher bitrate helps with "smearing", so it could possibly improve detection if your confidence scores are low or you are missing detections altogether. A fine-tuned Frigate+ model will probably improve person detection, especially if you've submitted many examples.
